### PR TITLE
Added a nav-bar to the header section

### PR DESCRIPTION
**Pull request by SilvernusIfoghale**
Updated the header section of the index.html file

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,20 +1,50 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
-</head>
-<body>
-    <header><!-- Todo1: Silvernus, add the navbar here--> </header>
+  </head>
+  <body>
+    <header style="padding: 0px 20px">
```
Comment (sulenchy) [inline src/index.html:10]: This line looks good to me

**Review by sulenchy — CHANGES_REQUESTED** [review]
Please, fix the merge conflict on the PR

File: src/index.html [hunk 2]
```diff
@@ -1,20 +1,50 @@
...
+        <ul
+          style="
+            display: flex;
+            gap: 15px;
+            align-items: center;
+            list-style-type: none;
+          "
+        >
+          <li>
+            <a href="#">Home</a>
```
Comment (sulenchy) [inline src/index.html]:
```suggestion
            <a href="#home">Home</a>
```

**Comment by SilvernusIfoghale** [comment]
Fixed

File: src/index.html [hunk 3]
```diff
@@ -6,17 +6,52 @@
...
+            gap: 15px;
+            align-items: center;
+            list-style-type: none;
+          "
+        >
+          <li>
+            <a href="#home">Home</a>
+          </li>
+          <li>
+            <a href="#">Services</a>
```
Comment (sulenchy) [inline src/index.html:29]:
```suggestion
            <a href="#services">Services</a>
```

**Review by sulenchy — APPROVED** [review]
Thanks for the changes

**Pull request merged**